templates/feature/app/styles/components/_ads.scss

```diff
@@ -33,7 +33,7 @@
   }
 
   > a {
-    color: $color-tribune-blue;
+    color: #347d90; // bespoke teal for a11y
```

Comment:
Is this teal a variable in queso's color variables?

Reply:
It isn't, but now that I think about it, it for sure should be. Color variables are free (as in don't cost us any additional output) so it will only help! I'll patch that real quick on the queso side and version this up to reflect. Good question

Reply:
This is done @mandicai 👍